### PR TITLE
streamlit app v2 with new data

### PR DESCRIPTION
**Pull request by scorreia**
The new streamlit app uses the new data generated by the "questionnaire" team. 
This PR also contains several notebooks for testing purpose.

### CODE REVIEW

File: app/README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Pour exécuter l'application Streamlit de , lancer la commande suivante depuis ce répertoire : 
-    `streamlit run odi_streamlit.py`
+Pour exécuter l'application Streamlit de , lancer la commande suivante depuis le répertoire parent :
```
Comment (kaaloo) [inline app/README.md:1]: Would be great to have an invoke task for this, something like `invoke streamlit`.

**Review by kaaloo — CHANGES_REQUESTED** [review]
Beautiful results!  🤯 💯 I added a suggestion, not critical,  but there seems to be an encoding error in the data:

image

File: app/README.md (hunk 1 above)
Comment (scorreia) [reply app/README.md:1]: Thanks. Invoke command added.

Comment (scorreia) [reply app/README.md:1]: we'll fix the encoding issue in  a later PR.  We still have some work to do on the data.

**Pull request merged**